geopyspark/geotrellis/__init__.py

```diff
@@ -234,6 +234,45 @@ def _asdict(self):
                     'proj4': self.proj4}
 
 
+GlobalLayout = namedtuple("GlobalLayout", 'tile_size zoom threshold')
+"""
+TileLayout type that spans global CRS extent.
+When passed in place of LayoutDefinition it signifies that a LayoutDefinition instance 
+should be constructed such that it fits the global CRS extent.
+The cell resolution of resulting layout will be one of resolutions implied by power of 2 pyramid for that CRS.
+Tiling to this layout will likely result in either up-sampling or down-sampling the source raster.
+
+Args:
+    tile_size (int): The number of columns and row pixels in each tile.
+    zoom (int, optional): Override the zoom level in power of 2 pyramid.
+    threshold(float, optional): The percentage difference between a cell size and a zoom level
+                                and the resolution difference between that zoom level and the next
+                                that is tolerated to snap to the lower-resolution zoom level.
+                                For example, if this paramter is 0.1, that means we're willing to downsample
+                                rasters with a higher resolution in order to fit them to some zoom level Z,
+                                if the difference is resolution is less than or equal to 10% the difference
+                                between the resolutions of zoom level Z and zoom level Z+1.
+
+Returns:
+    :obj:`~geopyspark.geotrellis.GlobalLayout`
+"""
+GlobalLayout.__new__.__defaults__ = (256, None, 0.1)
```

Comment:
`ZoomedLayoutScheme` is coded to always be square though. dealing with ingested non square tiles is a pretty good reason to expose tileCols, tileRows in `LocalLayout` for that case eventually when you read in already tiled dataset and you just need to give it a key.